onnx/test/test_external_data.py

```diff
@@ -342,49 +342,86 @@ def test_save_model_does_convert_and_saves_the_model(self):  # type: () -> None
         self.assertFalse(attribute_tensor.HasField("data_location"))
         self.assertTrue(np.allclose(to_array(attribute_tensor), self.attribute_value))
 
+    def test_save_model_does_not_load_external_tensor(self):  # type: () -> None
+        model_file_path = self.get_temp_model_filename()
+        onnx.save_model(self.model,
+                        model_file_path,
+                        save_as_external_data=True,
+                        location=None,
+                        size_threshold=0,
+                        convert_attribute=False)
+        # Save without load_external_data
+        model = onnx.load_model(model_file_path, load_external_data=False)
+        onnx.save_model(self.model,
+                        model_file_path,
+                        save_as_external_data=True,
+                        location=None,
+                        size_threshold=0,
+                        convert_attribute=False)
+        # Load the saved model again; Only works if the saved path is under the same directory
+        model = onnx.load_model(model_file_path)
 
-# The following test will fail in some platforms
-# because >2GB proto python object is not allowed
-# Disable it for now and it should be fixed after 1.8 Release
-"""
```

Comment:
Remove this test for good because in some environments loading model > 2GB will crash and it cannot be tested.